src/object_accessor.hpp

```diff
@@ -34,9 +34,7 @@
 #include <realm/util/assert.hpp>
 #include <realm/table_view.hpp>
 
-#if REALM_HAVE_SYNC_STABLE_IDS
```

Comment:
The things in non-sync-specific files guarded by REALM_HAVE_SYNC_STABLE_IDS need to be changed to REALM_ENABLE_SYNC rather than removed entirely to maintain support for building non-sync.

Reply:
Will do